debezium-connector-yugabytedb2/src/main/java/io/debezium/connector/yugabytedb/YugabyteDBConnector.java

```diff
@@ -350,6 +350,14 @@ else if (streamId == null || streamId.isEmpty()) {
                     continue;
                 }
 
+                // Ignore the tables without a pgschema_name, these tables are the ones created with the older versions of YugabyteDB where
```

Comment:
Ideally, we should just get the table for a namesapce/db from server side. Othewise it is very in-efficient to fetch tables from all the databases and then filter them.
Can you please file a ticket for the same.